bash_completion

```diff
@@ -2,6 +2,10 @@
 
 # bash completion for Node Version Manager (NVM)
 
+if ! nvm &> /dev/null; then
+    return
```

Comment:
Refer to the common bash_completions, I think it's fine to just silently do nothing, but we can print error message if you want.